straxen/plugins/high_energy_processing.py

```diff
@@ -0,0 +1,121 @@
+import strax
+import straxen
+import numpy as np
+
+export, __all__ = strax.exporter()
+
+# These are also needed in peaklets, since hitfinding is repeated
+HITFINDER_OPTIONS_he = tuple([
+    strax.Option(
+        'hit_min_amplitude_he',track=False,
+        default=tuple(np.ones(800, dtype=np.int16) * 15),
+        help='Minimum hit amplitude in ADC counts above baseline. '
+             'See straxen.hit_min_amplitude for options.'
+    )])
+
+
+@export
+@strax.takes_config(
+    *HITFINDER_OPTIONS_he)
+class PulseProcessingHe(straxen.PulseProcessing):
+    __version__ = '0.2.5'
+    rechunk_on_save = False
+    depends_on = 'raw_records_he'
+    provides = 'records_he'
+    data_kind = 'records_he'
+
+    def infer_dtype(self):
+        dtype = strax.record_dtype(strax.DEFAULT_RECORD_LENGTH)
+        return dtype
+
+    def setup(self):
+        self.hev_enabled=False
+        self.config['hit_min_amplitude'] = self.config['hit_min_amplitude_he']
+
+    def compute(self, raw_records_he, start, end):
+        result = super().compute(raw_records_he, start,end)
+        return result['records']
+
+    
+@export
+@strax.takes_config(
+
+    strax.Option(
+        'n_tpc_pmts_all',track=False,default=800
```

Comment:
Perhaps you also want to change some of the options like min_area et cetera

Reply:
You can change them by setting min_area. When we'll have a new default value we should add them

Reply:
yeah, I meant that if we know anyway that the parameters should be different it's maybe easier to add them now (and make educated guesses like for the coincidence requirement which would be ~2 times lower since there are twice as little HE channels)